bundle/store.go

```diff
@@ -447,7 +448,7 @@ func erasePolicies(ctx context.Context, store storage.Store, txn storage.Transac
 		}
 		deleted := false
 		for root := range roots {
-			if strings.HasPrefix(path, root) {
+			if isContainedByDirectory(root, path) {
```

Comment:
This looks suspiciously similar to `RootPathsOverlap` or `RootPathsContain`, https://github.com/open-policy-agent/opa/blob/v0.33.1/bundle/bundle.go#L1033-L1072 -- Could we somehow use those helpers here?

Reply:
Nice I missed that when I was looking through the code. I think that should work perfectly.